generators/client/templates/angular/src/main/webapp/app/layouts/navbar/navbar.component.ts.ejs

```diff
@@ -30,6 +30,7 @@ import { LANGUAGES } from 'app/config/language.constants';
 import { AccountService } from 'app/core/auth/account.service';
 import { LoginService } from 'app/login/login.service';
 import { ProfileService } from 'app/layouts/profiles/profile.service';
+import * as dayjs from 'dayjs';
```

Comment:
I think this should be only imported when translations are enabled

Reply:
Yes i see @atomfrede, Roger that

Reply:
We agree that we need to do this for all front technologies @atomfrede ?

Reply:
For all other frontend technologies it is not required as the (service/reducer) files are only generated when i18n is enabled, so that should be ok.